src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -849,7 +849,7 @@ bool NetworkOPsImp::haveLedger (std::uint32_t seq)
 
 std::uint32_t NetworkOPsImp::getValidatedSeq ()
 {
-    return m_ledgerMaster.getValidatedLedger ()->getLedgerSeq ();
+    return m_ledgerMaster.getValidLedgerIndex ();
```

Comment:
It's not really a failure result. Zero means the only ledger we know is the genesis ledger. The typical check is whether a particular ledger sequence is before or after this returned value.